Library/ContextStorageAdapters/ContextStorageAdapterInterface.php

```diff
@@ -1,20 +1,37 @@
 <?php
 
+/**
+ * ContextStorageAdapterInterface
+ *
+ * @category ContextStorageAdapters
+ * @package  Disturb\ContextStorageAdapters
+ * @author   Alexandre DEFRETIN <adefretin@voyageprive.com>
+ * @license  https://github.com/vpg/disturb/blob/poc/LICENSE MIT Licence
+ * @version  0.1.0
+ * @link     http://example.com/my/bar Documentation of Foo.
+ */
+
 namespace Vpg\Disturb\ContextStorageAdapters;
 
 use \Phalcon\Config;
 
+
 /**
  * Interface ContextStorageAdapterInterface
  *
- * @package Disturb\ContextStorageAdapters
+ * @category ContextStorageAdapters
+ * @package  Disturb\ContextStorageAdapters
+ * @author   Alexandre DEFRETIN <adefretin@voyageprive.com>
+ * @license  https://github.com/vpg/disturb/blob/poc/LICENSE MIT Licence
+ * @version  0.1.0
+ * @link     http://example.com/my/bar Documentation of Foo.
  */
 interface ContextStorageAdapterInterface
```

Comment:
in my opinion it's overnaming, it should be named Interface, as the namespace already name it ContextStorageAdapters